resources/service-manager-proxy/README.md

```diff
@@ -0,0 +1,73 @@
+# service-broker-proxy-k8s
```

Comment:
we have some guideline I think to have it equal as chart name so in this case: `Service Manager Proxy`